tests/spec/core/linter-rules/check-internal-slots-spec.js

```diff
@@ -19,41 +19,45 @@ describe("Core Linter Rule - 'check-internal-slots'", () => {
     }
   });
 
-  it("only returns an error when there is no '.' between var and an internal slot", async () => {
+  it("returns an error when there is no '.' between var and an internal slot", async () => {
     doc.body.innerHTML = `
-    <var>bar</var><a>[[foo]]</a>
-    <var>bar</var>..<a>[[foo]]</a>
-    <var>bar</var> . <a>[[foo]]</a>
-    <var>bar</var> <a>[[foo]]</a>
-    <a>[[foo]]</a><var>bar</var>
-    <var>bar</var>.<a>[foo]</a>
-    <var>bar</var>.<a>foo</a>
-    <var>bar</var>.<a>[[foo</a>
-    <var>bar</var>.<a>foo]]</a>
-    <var></var>.<a></a>
-    <var></var><a></a>
-    <var>[[foo]]</var>.<a></a>
-    <var>[[foo]]</var>.<a>bar</a>
-    <var>bar</var>.<a>[[f oo]]</a>
+      <var>bar</var><a>[[foo]]</a>
+      <var>bar</var>.<a>[foo]</a>
+      <var>bar</var>.<a>foo</a>
+      <var>bar</var>.<a>[[foo</a>
+      <var>bar</var>.<a>foo]]</a>
+      <var></var>.<a></a>
+      <var></var><a></a>
+      <var>[[foo]]</var>.<a></a>
+      <var>[[foo]]</var>.<a>bar</a>
+      <var>bar</var>.<a>[[f oo]]</a>
     `;
     const results = await rule.lint(config, doc);
     expect(results.length).toEqual(1);
 
     const [result] = results;
     expect(result.name).toEqual(ruleName);
-    expect(result.occurrences).toEqual(13);
+    // first fails the isPrevVar check, rest fail isInternalSlot check
+    expect(result.occurrences).toEqual(10);
```

Comment:
Sorry, I may have confused you. This should only equal 1 (`<var>bar</var><a>[[foo]]</a>`). The `isInternalSlot` check doesn't count as this (and all the others are fine):

```
... the <var>foo</var>. The internal slot <a>[[bar]]</a>...
```

In specs, you can have `[[foo bar]]` for whatever reason (similar for the rest of the test conditions above). They are all valid, except for `<var>bar</var><a>[[foo]]</a>`.